pywbem/_cim_operations.py

```diff
@@ -9796,6 +9977,61 @@ def DeleteQualifier(self, QualifierName, namespace=None):
             if self._operation_recorders:
                 self.operation_recorder_stage_result(None, exc)
 
+    def ExportIndication(self, NewIndication):
+        # pylint: disable=invalid-name
+        """
+        Send an indication to a WBEM listener.
+
+        This method performs the ExportIndication export operation
```

Comment:
I think we have one too many exports here.  This method performs the ExportIndication operation ...

Reply:
It is an "export operation". DSP0200 uses "operation" for operations targeting a server, and "export operation" for operations originating on a server.

Reply:
Since that is the spec, I respect that.  Suggest simply note that this is as defined in DSP0200.  I think in a lot of places we include the word request for other operations which means we are wrong there I would guess.

Reply:
It already says "see DSP0200".

Reply:
Double checked with DSP0200 ans it actually uses the term "export method". Changed all occurrences of "export operation" to "export method".